### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by AlissonRoss**
Super barebones/simple 404 page for now

### CODE REVIEW

**Review by nlawler1737 — APPROVED** [review]
looks good!

**Review by lclemente225 — DISMISSED** [review]
Looks good

**Review by nlawler1737 — CHANGES_REQUESTED** [review]
The path to broken.png should be "../../assets/broken.png".

**Comment by AlissonRoss** [comment]
Updated the 404 Page to the changes in main and added it to the sidebar and router. 

iPhone 11 Pro 
image
Galaxy Note 20
image
Desktop:
image

**Comment by AlissonRoss** [comment]
> The path to broken.png should be "../../assets/broken.png".

This should be fixed now

File: src/timetracker/pages/404-page/404-page.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import BrokenRobot from '../../assets/broken.png'
+export default function PageNotFound() {
+    return (
+        <div className="min-h-screen body text-white">
+            <div className="flex flex-col items-center">
```
Comment (nlawler1737) [inline src/timetracker/pages/404-page/404-page.jsx:5]: I would also add text-center so it doesn't default to left aligned on smaller screens

**Review by nlawler1737 — DISMISSED** [review]
looks good

**Review by lclemente225 — DISMISSED** [review]
Looks good, I just want to note that we should have a redirect in the config for this page when the page returns a 404 code.

**Review by nlawler1737 — APPROVED** [review]

File: src/timetracker/components/navbar/Sidebar.jsx
Comment (lclemente225) [inline src/timetracker/components/navbar/Sidebar.jsx:1]: Question: I noticed that the 404 page is listed in the sidebar. Will it be there on deployment?

File: src/main.jsx [hunk 2]
```diff
@@ -67,6 +69,10 @@ const router = createBrowserRouter([
             path: "/confirmresetpassword",
             element: <ConfirmPasswordReset />,
           },
+          {
+            path: "/*",
+            element: <PageNotFound/>
+          },
```
Comment (lclemente225) [inline src/main.jsx:75]: Question: I don't have experience with Browser Router, but I was wondering if this path of "/*" for the PageNotFound component was fine to use.

**Review by lclemente225 — APPROVED** [review]
Looks good

File: src/timetracker/components/navbar/Sidebar.jsx
Comment (AlissonRoss) [reply src/timetracker/components/navbar/Sidebar.jsx:1]: I added it to main.jsx as a route. Hopefully this helps

**Pull request merged**